packages/core/src/config.ts

```diff
@@ -100,6 +100,7 @@ export interface MedplumInfraConfig {
   rdsInstances: number;
   rdsInstanceType: string;
   rdsSecretsArn?: string;
+  postgresVersion?: string;
```

Comment:
Can we add this to `MedplumSourceInfraConfig` as well?

```ts
postgresVersion?: ValueOrExternalSecret<string>;
```

Reply:
How would you feel about `rdsInstanceVersion` for naming consistency?

I know this is used inside of `rds.DatabaseClusterEngine.auroraPostgres`, so it currently is only used for Postgres.  If we ever supported other RDS databases in the future, then this name becomes wrong or ambiguous.